zephyr build: Unknown CMake command "zephyr_get_include_directories_for_lang_as_string"

**Describe the bug**
Failed build process:

`Unknown CMake command "zephyr_get_include_directories_for_lang_as_string"`

**To Reproduce**
Zephyr build, with a recent version of zephyr: https://github.com/zephyrproject-rtos/zephyr

 ```
 lorenzo@MacBook-Air-di-Lorenzo ~/gitHubRepo/recentAnyLedgerRecentZephyr $ cd zephyr/
 lorenzo@MacBook-Air-di-Lorenzo ~/gitHubRepo/recentAnyLedgerRecentZephyr/zephyr $ export GNUARMEMB_TOOLCHAIN_PATH="~/opt/gcc-arm-none-eabi-7-2018-q2-update/"
 lorenzo@MacBook-Air-di-Lorenzo ~/gitHubRepo/recentAnyLedgerRecentZephyr/zephyr $ export ZEPHYR_TOOLCHAIN_VARIANT=gnuarmemb
 lorenzo@MacBook-Air-di-Lorenzo ~/gitHubRepo/recentAnyLedgerRecentZephyr/zephyr $  source zephyr-env.sh
 lorenzo@MacBook-Air-di-Lorenzo ~/gitHubRepo/recentAnyLedgerRecentZephyr/zephyr $ cd samples/anyledger/anyledger-wallet/
 lorenzo@MacBook-Air-di-Lorenzo ~/gitHubRepo/recentAnyLedgerRecentZephyr/zephyr/samples/anyledger/anyledger-wallet $ 
 lorenzo@MacBook-Air-di-Lorenzo 
 lorenzo@MacBook-Air-di-Lorenzo ~/gitHubRepo/recentAnyLedgerRecentZephyr/zephyr/samples/anyledger/anyledger-wallet $ mkdir build
 lorenzo@MacBook-Air-di-Lorenzo ~/gitHubRepo/recentAnyLedgerRecentZephyr/zephyr/samples/anyledger/anyledger-wallet $ cd build
 lorenzo@MacBook-Air-di-Lorenzo ~/gitHubRepo/recentAnyLedgerRecentZephyr/zephyr/samples/anyledger/anyledger-wallet/build $ cmake -GNinja -DBOARD=nrf52840_pca10056 -DBUILD_XCOMPILE=1 ../

```

**Desktop (please complete the following information):**
 - OS: macOS


**Additional context**
```
 lorenzo@MacBook-Air-di-Lorenzo ~/gitHubRepo/recentAnyLedgerRecentZephyr/zephyr/samples/anyledger/anyledger-wallet/build $ cmake -GNinja -DBOARD=nrf52840_pca10056 -DBUILD_XCOMPILE=1 ../
-- Found PythonInterp: /Users/lorenzo/anaconda/bin/python3 (found suitable version "3.6", minimum required is "3") 
-- The C compiler identification is AppleClang 9.1.0.9020039
-- The CXX compiler identification is AppleClang 9.1.0.9020039
-- Check for working C compiler: /Library/Developer/CommandLineTools/usr/bin/cc
-- Check for working C compiler: /Library/Developer/CommandLineTools/usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /Library/Developer/CommandLineTools/usr/bin/c++
-- Check for working CXX compiler: /Library/Developer/CommandLineTools/usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at cmake/zephyr.cmake:7 (zephyr_get_include_directories_for_lang_as_string):
  Unknown CMake command "zephyr_get_include_directories_for_lang_as_string".
Call Stack (most recent call first):
  CMakeLists.txt:18 (include)


-- Configuring incomplete, errors occurred!
See also "/Users/lorenzo/gitHubRepo/recentAnyLedgerRecentZephyr/zephyr/samples/anyledger/anyledger-wallet/build/CMakeFiles/CMakeOutput.log".
```

--- a/README.md
+++ b/README.md
@@ -20,14 +20,28 @@ Have a look at our [contribution guidelines](https://github.com/AnyLedger/anyled
 ### Dependencies
 
 ##### Linux (Ubuntu)
+On a fresh Ubuntu:18.04 docker image. Adapt as appropriate to your distro.
 ```
-apt-get install libgtest-dev g++ make cmake wget build-essential clang clang-tidy ninja-build git python3-pip python-protobuf -y
+apt-get update
+apt-get install libgtest-dev g++ make cmake wget build-essential clang clang-tidy ninja-build git python3-pip protobuf-compiler python-protobuf -y
 
 # Ubuntu's libgtest-dev is only the source. You have to compile them manually.
-cd /usr/src/gtest; cmake .; cmake --build . --target install
+cd /usr/src/gtest; cmake .; cmake --build . --target install 
 
+# Install Ethereum Solidity compiler
+apt-get install software-properties-common -y && add-apt-repository ppa:ethereum/ethereum -y && apt-get update
+apt-get install solc -y
+
+# without this, you will get a Python 3 ASCII/UTF8 error on Ubuntu
+export LC_ALL=C.UTF-8
+export LANG=C.UTF-8
+cd /
+git clone https://github.com/AnyLedger/anyledger-wallet
+
+alias build='rm -rf build; mkdir build; cd build; cmake -GNinja ../; ninja; cd ..'
+
+cd anyledger-wallet; pip3 install --user -r requirements.txt; build
 ```
-Install [protobuf](https://askubuntu.com/questions/1072683/how-can-i-install-protoc-on-ubuntu-16-04)
 
 ##### macOS
 

--- a/cmake/nanopb.cmake
+++ b/cmake/nanopb.cmake
@@ -27,12 +27,8 @@ function(nanopb_compile input SRCS HDRS)
     # compiled file name w/o the suffix
     get_filename_component(PROTO_NAME ${input} NAME_WE)
     get_filename_component(PROTO_PATH ${input} DIRECTORY)
-    execute_process(
-        COMMAND ${PYTHON_EXECUTABLE} -c
-        "from distutils import sysconfig; print(sysconfig.get_python_lib(prefix='${BUILD_INSTALL_DIR}')+'/nanopb/');"
-            OUTPUT_VARIABLE NANOPB_INST_DIR
-            OUTPUT_STRIP_TRAILING_WHITESPACE
-    )
+    
+    set (NANOPB_INST_DIR ${mylib_source_dir}/generator)
     add_custom_command(
         WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}
         COMMAND protoc --plugin=${NANOPB_INST_DIR}/protoc-gen-nanopb

